docs/projects/blank/tutorial/0/what-it-does.md

```diff
@@ -0,0 +1,90 @@
+<%(TOC)%>
+
+# What the Blank Project does
+
+The Blank Project offers the basic ability to launch a deployment, connect a client-worker and server-worker to it, and let the Player Lifecycle module create a player entity to represent the client.
+
+On this page, we’ll walk you through this functionality, and introduce the tooling and workflows that you should become familiar with along the way.
+
+## Opening the project
+
+First, open the Unity Project found at `workers/unity/`. The first time you open the project, Unity will retrieve all the packages defined in the `manifest.json` and cache them on your machine. After resolving the packages, you’ll see a popup that says `Generating code`.
+
+<img src="{{assetRoot}}assets/blank/tutorial/0/generating-code.png" style="margin: 0 auto; width: 50%; display: block;" />
+
+## Launch a local deployment
+
+Once the project is ready, hit `Ctrl+L` on Windows, or `Cmd+L` on Mac, to launch a local deployment. This opens a new terminal window and launches a local instance of SpatialOS using the `default_launch.json` launch configuration.
```

Comment:
Add "(located in the root folder of the Blank Project)" after `default_launch.json`